configs/_base_/datasets/imagenet_rgb.py

```diff
@@ -16,7 +16,7 @@
 
 test_pipeline = [
     dict(type='LoadImageFromFile'),
-    dict(type='Resize', size=(256, -1), backend='pillow'),
+    dict(type='Resize', size=(-1, 256), backend='pillow'),
```

Comment:
Imagenet and CIFAR configs should not be changed because we use MMCls' dataset.

Reply:
Interesting. So if one uses dataset type like mmcls.DATASET, then all the pipelines will automatically be imported from the mmcls? I didn't know this feature, and I always tried to write a custom Compose to incorporate pipelines from different mmlab repository.

Reply:
We only support automatically importing for MMCls, and we do this in the following code:
https://github.com/open-mmlab/mmgeneration/blob/69333cfa4b41768b9d700d2a913e3cfa567e1c9e/mmgen/datasets/builder.py#L49-L57

You may modify this part if you want to import datasets from other openmmlab's repo.

Reply:
It seems only can import `dataset` from the `mmcls` when dataset type starting with `mmcls.`, but doesn't affect the pipelines automatically. And I just checked the `Compose` in mmgen:
https://github.com/open-mmlab/mmgeneration/blob/9697976e5f820338413b6e82a0526dfc3317b772/mmgen/datasets/pipelines/compose.py#L24-L38
it seems that the pipelines also need to have a prefix `mmcls.` to make themselves be imported from the mmcls? But in the config:
https://github.com/open-mmlab/mmgeneration/blob/9697976e5f820338413b6e82a0526dfc3317b772/configs/_base_/datasets/imagenet_rgb.py#L1-L24
These pipelines don't start with `mmcls.`. Are you sure they, e.g. `Resize`, will use the mmcls ones?

Reply:
`build_dataset_mmcls` build datasets with MMCls' `PIPELINE`.
Adding `mmcls` in MMGen's dataset pipeline means we can import a specific pipeline from MMCls.

Reply:
Got it now.